templates/init/functions/typescript/tslint.json

```diff
@@ -1,115 +0,0 @@
-    "adjacent-overload-signatures": true,
-
-    // Do not allow the subtle/obscure comma operator.
-    "ban-comma-operator": true,
-
-    // Do not allow internal modules or namespaces . These are deprecated in favor of ES6 modules.
-    "no-namespace": true,
-
-    // Do not allow parameters to be reassigned. To avoid bugs, developers should instead assign new values to new vars.
-    "no-parameter-reassignment": true,
-
-    // Force the use of ES6-style imports instead of /// <reference path=> imports.
-    "no-reference": true,
-
-    // Do not allow type assertions that do nothing. This is a big warning that the developer may not understand the
-    // code currently being edited (they may be incorrectly handling a different type case that does not exist).
-    "no-unnecessary-type-assertion": true,
-
-    // Disallow nonsensical label usage.
-    "label-position": true,
-
-    // Disallows the (often typo) syntax if (var1 = var2). Replace with if (var2) { var1 = var2 }.
-    "no-conditional-assignment": true,
-
-    // Disallows constructors for primitive types (e.g. new Number('123'), though Number('123') is still allowed).
-    "no-construct": true,
-
-    // Do not allow super() to be called twice in a constructor.
-    "no-duplicate-super": true,
-
-    // Do not allow the same case to appear more than once in a switch block.
-    "no-duplicate-switch-case": true,
-
-    // Do not allow a variable to be declared more than once in the same block. Consider function parameters in this
-    // rule.
-    "no-duplicate-variable": [true, "check-parameters"],
-
-    // Disallows a variable definition in an inner scope from shadowing a variable in an outer scope. Developers should
-    // instead use a separate variable name.
-    "no-shadowed-variable": true,
-
-    // Empty blocks are almost never needed. Allow the one general exception: empty catch blocks.
-    "no-empty": [true, "allow-empty-catch"],
-
-    // Functions must either be handled directly (e.g. with a catch() handler) or returned to another function.
-    // This is a major source of errors in Cloud Functions and the team strongly recommends leaving this rule on.
-    "no-floating-promises": true,
-
-    // Do not allow any imports for modules that are not in package.json. These will almost certainly fail when
-    // deployed.
-    "no-implicit-dependencies": true,
-
-    // The 'this' keyword can only be used inside of classes.
-    "no-invalid-this": true,
-
-    // Do not allow strings to be thrown because they will not include stack traces. Throw Errors instead.
-    "no-string-throw": true,
-
-    // Disallow control flow statements, such as return, continue, break, and throw in finally blocks.
-    "no-unsafe-finally": true,
-
-    // Expressions must always return a value. Avoids common errors like const myValue = functionReturningVoid();
-    "no-void-expression": [true, "ignore-arrow-function-shorthand"],
```

Comment:
.eslintrc.js doesn't have this "ignore-arrow-function-shorthand" flag on "no-void"  - will this change the behavior? Is there a way to add this to .eslintrc.js?

Reply:
According to [Migration Guide](https://github.com/typescript-eslint/typescript-eslint/blob/96342e328652e32f44f2e87b62e3d5b4f6dbd73f/packages/eslint-plugin/ROADMAP.md#functionality), there seems to be no alternative.

Reply:
Ahhh, i see now that no-void and no-void-expression are not related. Regardless, i'm ok with this change. 

Thanks for linking the migration guide too, that is very useful.